collect_app/src/main/res/layout/empty_list_view.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="@dimen/margin_standard"
+    android:background="?colorSurfaceContainerLow">
+
+    <ImageView
+        android:id="@+id/icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:alpha="@dimen/medium_emphasis"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/title"
+        app:layout_constraintVertical_chainStyle="packed"
+        tools:src="@drawable/ic_baseline_edit_72" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textAppearance="?textAppearanceLabelExtraLarge"
+        android:fontFamily="sans-serif-medium"
```

Comment:
Is there another text appearance we could use instead of specifying a font here?

Reply:
Not really maybe `textAppearanceLabelExtraLarge` should use that font family by default? `textAppearanceLabelLarge` does so `textAppearanceLabelExtraLarge` should as well I guess?

Reply:
> Not really but maybe textAppearanceLabelExtraLarge should use that font family by default?

Maybe! I do wonder if we should be using one of the title tokens here rather than a label one as they're typically for controls ([docs here](https://m3.material.io/styles/typography/applying-type#af6eb002-9cbb-4b64-bce6-1315d2252364)).